gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Mon Oct 24 17:10:57 PDT 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
```

Comment:
7.6- the latest

Reply:
`7.5` is what was used in the last BOM update to the main branch. If I change it here we also have to change it there. @yenerm do you have thoughts on this?

Reply:
ok, let's keep 7.5